Makefile

```diff
@@ -48,6 +48,7 @@ IMAGES= \
     shell/public/wrench.png \
     shell/public/download.png \
     shell/public/key.png \
+	shell/public/share.png \
```

Comment:
seems your editor inserted a tab here where the surrounding lines use spaces (below for share-m.png, too)